modules/microshift-viewing-audit-logs.adoc

```diff
@@ -21,25 +21,28 @@ You can identify pod security admission violations on a workload by viewing the
 ----
 $ <node_name>=$(oc get node -ojsonpath='{.items[0].metadata.name}')
 ----
+//output example?
+
 . To view the audit logs, run the following command:
 +
 [source,terminal]
 ----
-$ oc adm node-logs <node_name> --path=kube-apiserver/
+$ oc adm node-logs <node_name> --path=kube-apiserver/ # <1>
```

Comment:
I don't know if this will cause an issue, but you don't need the `#` in a terminal block. You only need to comment them out if there is another line following (example [here](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#callouts-in-code-blocks)). If you have done this elsewhere and it works, feel free to ignore :sweat_smile: 

```suggestion
$ oc adm node-logs <node_name> --path=kube-apiserver/ <1>
```